### PR TITLE
Implemented Popup view modal for projects/works full view

### PR DESCRIPTION
**Pull request by harlexkhal**
- Implemented a pop-up modal that gets triggered on any of the project button click events.
-  Implemented keyframe animation on pop-up modal
- Changed CSS animation style on logo

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,7 +118,7 @@ <h2 class="project_title">Tonic</h2>
                   <li class="project_tag">javaScript</li>
                 </ul>
               </div>
-              <button type="button" class="view_project_btn">
+              <button type="button" class="view_project_btn proj_1">
                 See Project
               </button>
             </div>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Good job creating the dynamic code👍. Kindly note that already created HTML code for the projects list should be removed and all the HTML of the section is created when the page loads🙄

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👋, 
## Good job so far!👌
### Summary 👏
- No linter errors ✔
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] I'd suggest that your commit messages should always be in present tense `i.e` you should use `Add` instead of `Added`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (harlexkhal) [reply index.html]: hi @rotimiazeez ,Thanks for the review, but your message here is not clear on what changes you want me to make.

Comment (harlexkhal) [reply index.html]: This project only requires dynamicity of the Pop up modal window not the whole page @rotimiazeez

File: index.html [hunk 2]
```diff
@@ -359,6 +359,54 @@ <h2 class="contact_header">Contact me</h2>
...
+                      See Source <img src="images/see_source_btn.png" alt="hamburger menu" class="btn_icon"/>
+                    </button>
+                  </form>
+                </li>
+               </ul>
+            </div>
+          </div>
+        </div>
+      </div>    
+    </div>
```
Comment (descholar-ceo) [inline index.html]: - [ ] As it was stated in the previous review; the requirements were saying that you were supposed to remove all of these projects added manually in your HTML codes and rely on the ones you have in js, which will be rendered dynamically,
Screenshot from 2022-01-26 01-09-13

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @harlexkhal , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (harlexkhal) [reply index.html]: Thanks @descholar-ceo for clearing this out. i've just implemented the changes.

**Review by julie-ify — APPROVED** [review]
## Hi @harlexkhal, 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**